### DISCUSSION TITLE
turn off function detail popup

### DISCUSSION THREAD

**Discussion by stevenlis** [microsoft/pylance-release#4156] [Q&A] [answered]
Hi guys. I have a question of how to turn off this popup since it appears every time my cursor is in a function, which blocks my code up.

![CS-UeWTLhbg@2x](https://user-images.githubusercontent.com/33796896/228039921-5c2f55e8-4a68-4215-bd97-e12c2a047743.png)

**Comment by bschnurr** [comment, marked as answer]
there is a vscode setting for it.

add this to your settings.json

```json
"editor.parameterHints.enabled": false
```

**Comment by stevenlis** [comment]
@bschnurr Thanks a lot man!!!
I wonder if I missed something here. Is there a way to only toggle off the popup while my cursor is in the function()
If I put my cursor on the function name, I still wish to see it, but right now, i can only do that by using my mouse to hover over the function name.

<img width="175" alt="CS-w46tLmh0@2x" src="https://user-images.githubusercontent.com/33796896/228070088-cc2bb576-4f0e-4bd7-ada3-8e3c94d5c1c5.png">
